doc/rst/source/gmtconvert.rst

```diff
@@ -116,11 +116,11 @@ Optional Arguments
 
 .. _-F:
 
-**-F**\ [**c**\|\ **n**\|\ **r**\|\ **v**][**a**\|\ **f**\|\ **s**\|\ **r**\|\ *refpoint*]
+**-F**\ [**c**\|\ **n**\|\ **p**\|\ **v**][**a**\|\ **f**\|\ **s**\|\ **r**\|\ *refpoint*]
```

Comment:
Is this a documentation fix or something that's being changed in the source code as part of this PR? I haven't build this branch, but am unable to get -Fp to work (working from the base code in (test/psxy/qoutedlinedash.sh)

Reply:
I decided it was dumb to use **r** twice in this syntax so I switch r for refpoint to **p** for refpoint.  I changed the code to honor either choice.  I did not test anything though but it was a trivial change so I assume it works - in this branch.